src/storage/txn/scheduler.rs

```diff
@@ -726,6 +726,17 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
                 response_policy,
             }) => {
                 SCHED_STAGE_COUNTER_VEC.get(tag).write.inc();
+                match ctx.get_disk_full_opt() {
+                    DiskFullOpt::AllowedOnAlreadyFull => {
+                        to_be_write.disk_full_opt = DiskFullOpt::AllowedOnAlreadyFull
+                    }
+                    DiskFullOpt::AllowedOnAlmostFull => {
+                        if to_be_write.disk_full_opt != DiskFullOpt::AllowedOnAlreadyFull {
```

Comment:
How can it happen? If it can't happen theoretically, please use `debug_assert` or `assert`.

Reply:
Like Delete from table? TiDB set it to AllowedOnAlmostFull, but TiKV trait it as normal,.Then here we need change it.

Reply:
Got. Please comment it.

Reply:
Done